.github/actions/spelling/allow.txt

```diff
@@ -77,6 +77,7 @@ dragleave
 dragover
 endcapture
 extname
+fileupload
```

Comment:
Instead of adding these rare exceptions, use the new `#spell-check-ignore-line` pattern at the end of the affected lines to exclude them inline.